src/components/WwwFrame/PromotionalBanner/Banners/DepositIncentiveBanner.vue

```diff
@@ -1,4 +1,4 @@
-<template>
+<template v-if="!isLoggedin || !hasPromoBalance">
```

Comment:
Is the `v-if` on the component `template` working locally? I don't think that's a supported feature of Vue. There probably needs to be a new nested `template` with the `v-if`.